G2P.pm

```diff
@@ -40,56 +40,55 @@ limitations under the License.
+ af_from_vcf           : set value to 1 to include allele frequencies from VCF file. 
+                         Specifiy the list of reference populations to include with --af_from_vcf_keys    
+ af_from_vcf_keys      : reference populations used for annotating variant alleles with observed
+                         allele frequencies. Allele frequencies are retrieved from VCF files. If
+                         af_from_vcf is set to 1 but no populations specified with --af_from_vcf_keys
+                         all available reference populations are included. 
+                         TOPmed: TOPMed
+                         UK10K: ALSPAC, TWINSUK
+                         gnomAD exomes: gnomADe:AFR, gnomADe:ALL, gnomADe:AMR, gnomADe:ASJ, gnomADe:EAS, gnomADe:FIN, gnomADe:NFE, gnomADe:OTH, gnomADe:SAS
+                         gnomAD genomes: gnomADg:AFR, gnomADg:ALL, gnomADg:AMR, gnomADg:ASJ, gnomADg:EAS, gnomADg:FIN, gnomADg:NFE, gnomADg:OTH
+                         Separate multiple values with '&'
+ default_af            : default frequency of the input variant if no frequency data is
+                         found (0). This determines whether such variants are included;
+                         the value of 0 forces variants with no frequency data to be
+                         included as this is considered equivalent to having a frequency
+                         of 0. Set to 1 (or any value higher than af) to exclude them.
+ types                 : SO consequence types to include. Separate multiple values with '&'
+                         (splice_donor_variant,splice_acceptor_variant,stop_gained,
+                         frameshift_variant,stop_lost,initiator_codon_variant,
+                         inframe_insertion,inframe_deletion,missense_variant,
+                         coding_sequence_variant,start_lost,transcript_ablation,
+                         transcript_amplification,protein_altering_variant)
   
-  log_dir        : write stats to log files in log_dir 
+  log_dir              : write stats to log files in log_dir 
 
-  txt_report     : write all G2P complete genes and attributes to txt file
+  txt_report           : write all G2P complete genes and attributes to txt file
 
-  html_report    : write all G2P complete genes and attributes to html file
+  html_report          : write all G2P complete genes and attributes to html file
 
  Example:
 
- --plugin G2P,file=G2P.csv.gz,af_monoallelic=0.05,af_keys=AA&gnomAD_ASJ,types=stop_gained&frameshift_variant
- --plugin G2P,file=G2P.csv.gz,af_monoallelic=0.05,types=stop_gained&frameshift_variant
- --plugin G2P,file=G2P.csv.gz,af_monoallelic=0.05,af_from_vcf=1
- --plugin G2P,file=G2P.csv.gz
+ --plugin G2P,file=G2P.csv,af_monoallelic=0.05,af_keys=AA&gnomAD_ASJ,types=stop_gained&frameshift_variant
+ --plugin G2P,file=G2P.csv,af_monoallelic=0.05,types=stop_gained&frameshift_variant
+ --plugin G2P,file=G2P.csv,af_monoallelic=0.05,af_from_vcf=1
+ --plugin G2P,file=G2P.csv
```

Comment:
Just to highlight this as we might have some users queries if they used compressed files in the past

Reply:
There will be an error message if people try passing a zipped file